plugins/glustershd/glustershd.go

```diff
@@ -0,0 +1,102 @@
+package glustershd
+
+import (
+	"bytes"
+	"fmt"
+	"net"
+	"os/exec"
+	"path"
+
+	"github.com/cespare/xxhash"
+	"github.com/gluster/glusterd2/glusterd2/gdctx"
+
+	config "github.com/spf13/viper"
+)
+
+const (
+	glustershdBin = "glusterfs"
+)
+
+// Glustershd type represents information about Glustershd process
+type Glustershd struct {
+	binarypath     string
+	args           string
+	socketfilepath string
+	pidfilepath    string
+}
+
+// Name returns human-friendly name of the glustershd process. This is used for
+// logging
+func (b *Glustershd) Name() string {
```

Comment:
The variable `b` was named so because it was **b**rick. You can name it shd here instead.